addons/website_hr_recruitment/static/src/js/website_hr_applicant_form.js

```diff
@@ -0,0 +1,146 @@
+/** @odoo-module **/
+
+import publicWidget from "@web/legacy/js/public/public_widget";
+
+publicWidget.registry.hrRecruitment = publicWidget.Widget.extend({
+    selector : '#hr_recruitment_form',
+
+    events: {
+        'click #apply-btn': '_onClick',
```

Comment:
Maybe we could use a more specific function name for future use, such as `_onClickApplyButton`